### PR TITLE
Osdocs-4240 Added "only if you are using non-STS." to init command info for ROSA

### PR DESCRIPTION
**Pull request by eohartman**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OSDOCS-4240

Link to docs preview:
https://53482--docspreview.netlify.app/openshift-rosa/latest/rosa_cli/rosa-get-started-cli.html#rosa-initialize_rosa-getting-started-cli

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by eohartman** [comment]
Worth noting: this included fixing a typo elsewhere and another change request.

**Review by skrthomas — COMMENTED** [review]
/lgtm

File: modules/storage-expanding-add-volume-expansion.adoc [hunk 1]
```diff
@@ -12,7 +12,14 @@ have the `allowVolumeExpansion` field set to `true`.
 
 .Procedure
 
-* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute.
+* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute by running the following command:
```
Comment (bergerhoffer) [inline modules/storage-expanding-add-volume-expansion.adoc:15]: Just checking - is the update to this file something you were expecting, or is it an accidental inclusion from someone else's? It doesn't seem related to your other update, and the commit is shared between you and courtney.

**Review by bergerhoffer — COMMENTED** [review]
@eohartman A question on this update before we merge!

File: modules/storage-expanding-add-volume-expansion.adoc (hunk 1 above)
Comment (eohartman) [reply modules/storage-expanding-add-volume-expansion.adoc:15]: Accidental inclusion. Thanks for finding it.

**Review by bergerhoffer — COMMENTED** [review]
LGTM now, merging!
